scripts/zkai_attention_kv_snark_statement_receipt_gate.py

```
@@ -0,0 +1,840 @@
+            "statement",
+            "statement_commitment",
+            "verification_key",
+        },
+        "receipt",
+    )
+    expect_equal(receipt.get("schema"), RECEIPT_SCHEMA, "receipt schema")
+    expect_equal(receipt.get("non_claims"), NON_CLAIMS, "non claims")
+    proof, public_signals, verification_key = _snarkjs_payloads(receipt)
+    statement = require_object(receipt.get("statement"), "statement")
+    if receipt.get("statement_commitment") != statement_commitment(statement):
+        raise AttentionKvSnarkReceiptError("statement_commitment mismatch", layer="statement_commitment")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)